mass_mailing_switzerland/models/mass_mailing_campaign.py

```diff
@@ -27,7 +27,7 @@ class MassMailingCampaign(models.Model):
         'correspondence', related='campaign_id.correspondence_ids'
     )
     invoice_ids = fields.One2many(
-        'account.invoice', related='campaign_id.invoice_ids'
+        'account.invoice.line', related='campaign_id.invoice_ids'
```

Comment:
related field would become `campaign_id.invoice_line_ids` when the field in utm will be renamed as well.